eks-anywhere-common/Testers/Aqua/aqua-testJob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aqua-enforcer-tester
+  namespace: aqua
+spec:
+  template:
+    spec:
+      serviceAccountName: 'aqua-enforcer-sa'
+      containers:
+        - name: test-aqua-enforcer
+          image: 'alpine/k8s:1.26.2'
+          imagePullPolicy: Always
+          command:
+            - /bin/bash
+          args:
+            - '-c'
+            - >-
+              echo "1. Checking readiness probe"
+              aqua_enforcer_pod="";
+              while true;
+              do
+                aqua_enforcer_pod=$(kubectl get pod -n aqua -l app=aqua-enforcer-ds --field-selector=status.phase==Running -o jsonpath="{.items[0].metadata.name}{range .items[0].status.conditions[?(@.type=='Ready')]}");
```

Comment:
Checking the status of a pod run does not qualify for a functional test. Please check our [functional job requirements](https://aws-samples.github.io/eks-anywhere-addons/#functional-job-requirements) for detailed requirements. You can also check our Partner submission test jobs for more reference.